rest/model/task_annotations.go

```diff
@@ -190,3 +196,14 @@ func GetJiraTicketFromURL(URL string) (*thirdparty.JiraTicket, error) {
 
 	return nil, nil
 }
+
+func ValidateIssues(catcher grip.Catcher, issues []APIIssueLink) grip.Catcher {
```

Comment:
rather than passing in and returning a catcher, can ValidateIssues just create it's own catcher, and return a catcher.Resolve()?

Reply:
done